include/mbedtls/rsa.h

```diff
@@ -88,41 +89,41 @@ extern "C" {
  */
 typedef struct mbedtls_rsa_context
 {
-    int ver;                    /*!<  Reserved for internal purposes.
+    int MBEDTLS_PRIVATE(ver);                    /*!<  Reserved for internal purposes.
```

Comment:
Note: now that each field is explicitly marked as private, I think we should remove the `\note` above - direct manipulation is now more than deprecated, it's forbidden.